demo/main.js

```diff
@@ -302,8 +302,12 @@ class Demo extends Component {
   }
 
   raiseError(content, callback) {
-    window.alert(content);
-    callback?.();
+    this.notifyUser({
```

Comment:
I'd rather have a specific behaviour to handle `unhandledrejection` rather than completely changing the behaviour of raiseError altogether.

Reply:
Good point, the raiseError is used in business code. I'll just change the `unhandledrejection` and the `onError`.